cmd/server/main.go

```diff
@@ -4,30 +4,29 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net"
 
+	"go.viam.com/rdk/services/web"
 	"go.viam.com/rplidar"
 
-	_ "go.viam.com/rplidar/serial" //register
-
 	"github.com/edaniels/golog"
-	"go.uber.org/multierr"
-	"go.viam.com/core/config"
-	grpcserver "go.viam.com/core/grpc/server"
-	"go.viam.com/core/lidar/search"
-	pb "go.viam.com/core/proto/api/v1"
-	"go.viam.com/core/rlog"
-	robotimpl "go.viam.com/core/robot/impl"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/metadata/service"
+	"go.viam.com/rdk/rlog"
```

Comment:
You shouldn't need this AND golog (above) at the same time. This literally just pulls in golog. (This applies to the other files that do this as well.)

Reply:
In this case, `ContextualMain` requires a `golog.Logger`. I tried to recreate that using only the `rlog` module, e.g. by doing `rlog.golog.Logger`, but it wouldn't compile. I didn't find any examples that reflect solve it using only `rlog` or `golog`, either.

Do you know how to fix this, or if this is even possible without importing both `golog` and `rlog`?

Reply:
Fixed as discussed in chat; am using mix of both, but am making sure that all `golog.Global` are changed to `rlog.Logger`.

Done.